packages/core/src/App/AppContent.tsx

```diff
@@ -21,57 +18,32 @@ import Routes from './Containers/Routes/routes.jsx';
 import Devtools from './Devtools';
 import initDatadog from '../Utils/Datadog';
 import { ThemeProvider } from '@deriv-com/quill-ui';
+import { useGrowthbookFeatureFlag } from '@deriv/hooks';
 
 const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }) => {
     const store = useStore();
     const { has_wallet } = store.client;
 
+    const [isWebPasskeysFFEnabled, isGBLoaded] = useGrowthbookFeatureFlag({
+        featureFlag: 'web_passkeys',
+        defaultValue: false,
+    });
+    const [isServicePasskeysFFEnabled] = useGrowthbookFeatureFlag({
+        featureFlag: 'service_passkeys',
+        defaultValue: false,
+    });
     const isMounted = useIsMounted();
     const { data } = useRemoteConfig(isMounted());
-    const { marketing_growthbook, tracking_datadog, tracking_rudderstack, passkeys } = data;
+    const { tracking_datadog } = data;
     const is_passkeys_supported = browserSupportsWebAuthn();
 
-    const account_type = LocalStore?.get('active_loginid')
-        ?.match(/[a-zA-Z]+/g)
-        ?.join('');
-
     React.useEffect(() => {
-        if (process.env.RUDDERSTACK_KEY && tracking_rudderstack) {
-            const config = {
-                growthbookKey: marketing_growthbook ? process.env.GROWTHBOOK_CLIENT_KEY : undefined,
-                growthbookDecryptionKey: marketing_growthbook ? process.env.GROWTHBOOK_DECRYPTION_KEY : undefined,
-                rudderstackKey: process.env.RUDDERSTACK_KEY,
-            };
-            Analytics.initialise(config);
-            const ppc_campaign_cookies =
-                Cookies.getJSON('utm_data') === 'null'
-                    ? {
-                          utm_source: 'no source',
-                          utm_medium: 'no medium',
-                          utm_campaign: 'no campaign',
-                          utm_content: 'no content',
-                      }
-                    : Cookies.getJSON('utm_data');
-
-            Analytics.setAttributes({
-                account_type: account_type === 'null' ? 'unlogged' : account_type,
-                app_id: String(getAppId()),
-                device_type: store?.ui?.is_mobile ? 'mobile' : 'desktop',
-                device_language: navigator?.language || 'en-EN',
-                user_language: getLanguage().toLowerCase(),
-                country: Cookies.get('clients_country') || Cookies?.getJSON('website_status')?.clients_country,
-                utm_source: ppc_campaign_cookies?.utm_source,
-                utm_medium: ppc_campaign_cookies?.utm_medium,
-                utm_campaign: ppc_campaign_cookies?.utm_campaign,
-                utm_content: ppc_campaign_cookies?.utm_content,
-            });
+        if (isGBLoaded && isWebPasskeysFFEnabled && isServicePasskeysFFEnabled) {
```

Comment:
do we need to check BE flag service_passkeys?

Reply:
This is for extra safety. that in case any of the flags are off we should turn off immediately.

Reply:
@ali-hosseini-deriv , @amina-deriv , 
we agreed with BE team to use not the FF value, but check if FF is ON. these are different things.
I believe we should have one approach. 
for me FF value is more convenient, but we need to change it on BE side as well.
so let's use the check if FF is ON